app/components/app_session_needs_review_warning_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class AppSessionNeedsReviewWarningComponent < ViewComponent::Base
+  erb_template <<-ERB
+    <div class="nhsuk-warning-callout">
+      <h3 class="nhsuk-warning-callout__label">
+        <span role="text">
+          <span class="nhsuk-u-visually-hidden">Important: </span>
+          Needs review
+        </span>
+      </h3>
+
+      <ul>
+        <% if @session.patients.without_nhs_number.count > 0 %>
```

Comment:
```suggestion
        <% if patient_sessions.merge(Patient.without_nhs_number).exists? %>
```

Although do we need this line with the `render?` method?

Reply:
we don't need it